mewarpx/mewarpx/plotting.py

```diff
@@ -71,7 +71,7 @@ class ArrayPlot(object):
             },
```

Comment:
Should we put this file in `mewarpx/diags_store` or `mewarpx/utils_store` and rename it something like `plotting_util.py` since the module isn't directly used to affect the simulation, it's just to help plot diagnostics?